ambry-api/src/main/java/com.github.ambry/router/GetOperationType.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.router;
+
+public enum GetOperationType {
```

Comment:
I feel that this could simply be a public inner enum within `GetBlobOptions`. And this can simply be named as `OperationType` - one can simply deal with `GetBlobOptions.OperationType`. What do you think?